spec/util/miq-password_spec.rb

```diff
@@ -365,4 +376,22 @@ def with_key
   def erberize(password, passmethod = "MiqPassword")
     "<%= #{passmethod}.decrypt(\"#{password}\") %>"
   end
+
+  def with_random_key
```

Comment:
I was confused with the term "random" here. Maybe something like `with_external_key` or `with_alternate_key`?

It's not really random because if it were you wouldn't be able to assert on the encrypted value.